### DISCUSSION TITLE
Overriding Init-Containers?

### DISCUSSION THREAD

**Discussion by svelama** [devfile/api#1147] [Q&A]
I'm looking to override the security context for the **init-containers**(i.e. created by the DevWorkspace operator for cloning the repos), I can see that it's available for [pod and containers overrides](https://github.com/devfile/devworkspace-operator/blob/main/docs/additional-configuration.adoc#fine-grained-configuration-of-workspace-pods-and-containers), but I don't see any ways to override the init-containers security context, any suggestions/guidance is greatly appreciated.

**Comment by maysunfaisal** [comment]
@amisevsk would you be able to help with this question?

**Reply by amisevsk** [reply]
Container overrides should apply to init containers as well; if this is not the case, please report it as an issue!

**Reply by amisevsk** [reply]
For the project-clone container specifically -- that is managed by the DevWorkspace Operator, which does not provide a way to configure its security context directly. If there's a use case for configuring this security context, please file an issue in the DevWorkspace Operator repository (https://github.com/devfile/devworkspace-operator); we're generally reluctant to allow for configuration here as it would bring the assumptions DWO makes about project cloning into the "api" for the operator.
